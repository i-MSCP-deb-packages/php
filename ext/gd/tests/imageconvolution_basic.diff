001+ 7ac4be52c49a47db53d8e4ec3cfd29d8
001- 594576a2a2a689447ffc07eb5a73f09b