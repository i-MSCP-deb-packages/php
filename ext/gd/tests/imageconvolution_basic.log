
---- EXPECTED OUTPUT
594576a2a2a689447ffc07eb5a73f09b
---- ACTUAL OUTPUT
7ac4be52c49a47db53d8e4ec3cfd29d8
---- FAILED
