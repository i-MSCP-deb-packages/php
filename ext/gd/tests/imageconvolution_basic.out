7ac4be52c49a47db53d8e4ec3cfd29d8